PREREQUISITES.md

```diff
@@ -418,6 +422,9 @@ problems.
   you are using an old version of OpenSSL and need to clone the
   `openssl_1.1.1` branch of `intel-sgx-ssl` in the step above
 
+- If the message  `intel_sgx: SGX is not enabled` appears in `/var/log/syslog`
+  Intel SGX needs to be enabled in BIOS
```

Comment:
Is this a standard recommended way to detect and enable SGX ?

Reply:
No. It is only how to detect in Linux if SGX is enabled in BIOS.

This message appears even if SGX is supported by the processor an even if drivers are installed.